packages/dcos-integration-test/extra/test_ucr.py

```diff
@@ -0,0 +1,229 @@
+import uuid
+
+TEST_APP_NAME_FMT = 'integration-test-{}'
+
+
+def test_if_ucr_app_can_be_deployed_with_image_entrypoint(dcos_api_session):
+    """Marathon app deployment integration test using the Mesos Containerizer.
+
+    This test verifies that a marathon ucr app can execute a docker image
+    default entrypoint.
+    """
+    test_uuid = uuid.uuid4().hex
```

Comment:
I did think about that previously. Because the only parameters that I can include in `base_app` are  only 4: id, cpus, mem, instances. So I think it is fine to just explicitly define them for every app, which is also more clear to code reader.

To answer your question, the image default entrypoint will be executed if the docker image 'mesosphere/inky' is provided and no other cmd is specified. This is image default entrypoint (see docker image spec). 
https://github.com/docker/docker/blob/master/image/spec/v1.md

Mesos support those default entrypoint and cmd via docker runtime isolator, which is an important component of UCR.
https://github.com/apache/mesos/blob/master/docs/mesos-containerizer.md#determine-the-launch-command